web-ui-developer-puzzle-master/libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img class="reading-list-item--cover" [src]="b.coverUrl" [attr.alt] = "b.title + ' Cover'" />
```

Comment:
Actually hard to know this, but the alt tag should probably be empty in this case since there's an immediately adjacent text that describes the image content.

https://www.w3.org/WAI/tutorials/images/decision-tree/